vips/vip-319/bscmainnet.ts

```diff
@@ -0,0 +1,51 @@
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const vBNBAdmin_Implementation = "0xaA8D9558d8D45666552a72CECbdd0a746aeaCDc9";
+export const vBNBAdmin = "0x9A7890534d9d91d473F28cB97962d176e2B65f1d";
+export const ProxyAdmin = "0x6beb6D2695B67FEb73ad4f172E8E2975497187e4";
```

Comment:
I wonder if we could use consistent casing for addresses and contracts. I'm not consistent with this either – I usually use `UPPERCASE` for plain addresses but `bscmainnet.PascalCase` if the address is inside of some object (the latter corresponds to how our deployments are named). Well, except for [addresses in the VIP framework](https://github.com/VenusProtocol/vips/blob/main/src/networkAddresses.ts) because they are inside an object but they're in `UPPERCASE` 😅

I guess we can keep it as is in this PR, but if you have any thoughts please share.